Bump to 8.18.0, add discontinuation warning (#64)
elastic/enterprise-search-js: packages/enterprise-search-universal/README.md

# :warning: App Search and Workplace Search will be discontinued in 9.0

App Search and Workplace Search will be discontinued in 9.0.

> App Search and Workplace Search and the Enterprise Search clients will continue to be supported in their current form throughout 8.x versions, for as long as 8.x is supported.

Reviewer: ```suggestion
The App Search, Workplace Search and Enterprise Search clients will continue to be supported throughout the 8.x version lifecycle. 
```

A minor tweak, feel free to apply :)
